backupTool.py

```diff
@@ -0,0 +1,148 @@
+__author__ = 'marc'
+from constants import DATA_FOLDER
+import csv
+import db.datastore as db
+import errno
+import os
+import re
+import shutil
+import sqlite3 as lite
+import tarfile
+import time
+
+_TABLES = {
```

Comment:
think there might be a way to initialize this programatically out of the database schema? this seems like a maintenance hell.